File: VB/AjaxSupport/Controllers/HomeController.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.Mvc
Imports AjaxSupport.Models

Namespace AjaxSupport.Controllers
	Public Class HomeController
		Inherits Controller
		Public Function Index() As ActionResult
			Return View()
		End Function
		Public Function GridViewPartial() As ActionResult
			Return PartialView("GridViewPartial", InMemoryModel.GetDataTableModel())
		End Function
	End Class
End Namespace

File: CS/AjaxSupport/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;
using AjaxSupport.Models;

namespace AjaxSupport.Controllers {
    public class HomeController : Controller {
        public ActionResult Index() {
            return View();
        }
        public ActionResult GridViewPartial() {
            return PartialView("GridViewPartial", InMemoryModel.GetDataTableModel());
        }
    }
}